### DISCUSSION TITLE
batch upscale in runpod

### DISCUSSION THREAD

**Discussion by lgooders** [TheLastBen/fast-stable-diffusion#2224] [General]
Trying to batch upscale a decent size set but I have no clue what directory to add the photos to so they are uploaded to auto1111.  Keeps telling me no directory when i create one.  Noob question sorry.

**Comment by TheLastBen** [comment]
where do you have the images that you want to upscale ?

**Reply by lgooders** [reply]
locally

**Reply by TheLastBen** [reply]
you need to upload them

**Reply by lgooders** [reply]
where to?  Like I'm five please lol

**Comment by lgooders** [comment]
nm, im an idiot
